### PR TITLE
MHW_1: Eugene Syroezhkin

### PR DESCRIPTION
**Pull request by kaktusmm**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle).
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks.
- [x] my PR made from the fork feature branch to appropriate branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

File: src/homework.py [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
-nn: int = int(input())
-ll: int = int(input())
-dd = (((mm * 100) + nn) * ll)
-print(f'Cтоимость {ll} товаров = {dd // 100} рублей, {dd % 100} копеек')
+is_blocked = True
+is_registered = True
+
+
+has_permission = True if is_registered and not is_blocked else False
+print(has_permission)
```
Comment (kaktusmm) [inline src/homework.py:8]: круто

File: src/homework.py [hunk 2]
```diff
@@ -1,10 +1,8 @@
...
-"""
+is_admin = False
 
-mm: int = int(input())
-nn: int = int(input())
-ll: int = int(input())
-dd = (((mm * 100) + nn) * ll)
-print(f'Cтоимость {ll} товаров = {dd // 100} рублей, {dd % 100} копеек')
+is_blocked = True
+is_registered = True
```
Comment (kaktusmm) [inline src/homework.py:4]: переделать

**Review by mega-yujin — APPROVED** [review]
8

**Comment by mega-yujin** [comment]
ok

**Pull request merged**